examples/combobox/combobox-datepicker.html

```diff
@@ -0,0 +1,781 @@
+      <li><a href="https://github.com/w3c/aria-practices/projects/7">Related Issues</a></li>
+      <li><a href="../../#combobox">Design Pattern</a></li>
+    </ul>
+  </nav>
+  <main>
+    <h1>Date Picker Combobox Example</h1>
+    <p>
+      The below date picker demonstrates an implementation of the <a href="../../#combobox">combobox design pattern</a> that opens a dialog.
+      The date picker dialog is opened by activating the choose date button or by moving keyboard focus to the combobox and pressing <kbd>Down Arrow</kbd> or <kbd>Alt + Down Arrow</kbd>.
+      The  dialog contains an implementation of the <a href="../../#grid">grid pattern</a> for displaying a calendar and enabling selection of a date.
+      Additional buttons in the dialog are available for changing the month and year shown in the grid.
+    </p>
+    <p>Similar examples include:</p>
+    <ul>
+      <li><a href="../dialog-modal/datepicker-dialog.html">Date Picker Dialog example</a>: Demonstrates a dialog containing a calendar grid for choosing a date.</li>
+      <li><a href="combobox-select-only.html">Select-Only Combobox</a>: A single-select combobox with no text input that is functionally similar to an HTML <code>select</code> element.</li>
+      <li><a href="combobox-autocomplete-both.html">Editable Combobox with Both List and Inline Autocomplete</a>: An editable combobox that demonstrates the autocomplete behavior known as <q>list with inline autocomplete</q>.</li>
+      <li><a href="combobox-autocomplete-list.html">Editable Combobox with List Autocomplete</a>: An editable combobox that demonstrates the autocomplete behavior known as <q>list with manual selection</q>.</li>
+      <li><a href="combobox-autocomplete-none.html">Editable Combobox Without Autocomplete</a>: An editable combobox that demonstrates the behavior associated with <code>aria-autocomplete=none</code>.</li>
+      <li><a href="grid-combo.html">Editable Combobox with Grid Popup</a>: An editable combobox that presents suggestions in a grid, enabling users to navigate descriptive information about each suggestion.</li>
+    </ul>
+    
+  <section>
+    <h2 id="ex_label">Example</h2>
+
+    <div role="separator" id="ex_start_sep" aria-labelledby="ex_start_sep ex_label" aria-label="Start of"></div>
+    <div id="ex1">
+
+      <div id="myDatepicker"
+           class="combobox-datepicker">
+
+        <div class="label">
+          <label id="id-label-1" for="cb-textbox-1">
+            Date
+          </label>
+        </div>
+
+        <div class="group">
+          <input type="text"
+                id="cb-textbox-1"
+                aria-autocomplete="none"
+                role="combobox"
+                aria-expanded="false"
+                aria-haspopup="dialog"
+                aria-controls="cb-dialog-1"
+                aria-describedby="cb-description-1">
+
+          <span class="desc" id="cb-description-1">(date format: mm/dd/yy)</span>
```

Comment:
Date format needs to be 4-digit date, because if you type a 2-digit date, you'll end up back in the 20th century (i.e. in the 1900's). 😄 
```suggestion
          <span class="desc" id="cb-description-1">(date format: mm/dd/yyyy)</span>
```